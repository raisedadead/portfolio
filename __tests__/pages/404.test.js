import React from 'react';
import { render } from '@testing-library/react';
import FourOhFourMarkup from '../../src/pages/404';

describe(`404`, () => {
  it(`renders correctly`, () => {
    const {
      container: { firstChild }
    } = render(<FourOhFourMarkup />);
    expect(firstChild).toMatchSnapshot();
  });
});
