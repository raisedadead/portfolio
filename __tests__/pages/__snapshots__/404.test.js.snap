// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 renders correctly 1`] = `
<div
  class="css-17mhdrz elghiqq0"
>
  <div
    class="css-11inlkn elghiqq0"
  >
    <nav
      class="css-l60o26-PrimaryNav e1befqft0"
    >
      <a
        aria-label="Mrugesh Mohapatra's resumé"
        class="e1befqft1 e70ypka2 css-xlyo8k e19l42ga0"
        data-index="0"
        href="https://represent.io/mrugesh"
      >
        <span
          class="css-1xssqnp e70ypka1"
        >
          <span
            class="css-1sp7cuy e70ypka0"
          >
            <span
              class="css-1fns1v3 e1i5p1da0"
            >
              <span
                class="css-1h0pl34 ew65k9p0"
                id="tooltip-1"
              >
                <span
                  class="css-pusqig ew65k9p3"
                >
                  <span
                    aria-describedby="tooltip-1-content"
                    aria-owns="tooltip-1-content"
                    class="css-1myw1nk e3ztgjb0"
                    role="button"
                    tabindex="0"
                  >
                    resumé
                  </span>
                </span>
              </span>
            </span>
          </span>
        </span>
      </a>
      <a
        aria-label="Ask me anything"
        class="e1befqft1 e70ypka2 css-xlyo8k e19l42ga0"
        data-index="1"
        href="https://github.com/raisedadead/ama/"
      >
        <span
          class="css-1xssqnp e70ypka1"
        >
          <span
            class="css-1sp7cuy e70ypka0"
          >
            <span
              class="css-1fns1v3 e1i5p1da0"
            >
              <span
                class="css-1h0pl34 ew65k9p0"
                id="tooltip-2"
              >
                <span
                  class="css-pusqig ew65k9p3"
                >
                  <span
                    aria-describedby="tooltip-2-content"
                    aria-owns="tooltip-2-content"
                    class="css-1myw1nk e3ztgjb0"
                    role="button"
                    tabindex="0"
                  >
                    ama
                  </span>
                </span>
              </span>
            </span>
          </span>
        </span>
      </a>
    </nav>
  </div>
  <main>
    <main>
      <section
        class="css-11inlkn elghiqq0"
      >
        <div
          class="e7r6m60 css-wzh7ay elghiqq0"
        >
          <h2
            class="css-ov77vs e19le4zx0"
          >
            404 | Page not found.
          </h2>
          <p
            class="css-16qt340 e19le4zx0"
          >
            Ah..! That's my bad. Sorry. Why don't you checkout
             
            <a
              class="css-3r9jsu"
              href="/"
            >
              something here
            </a>
            .
          </p>
        </div>
      </section>
    </main>
  </main>
  <footer>
    <section
      class="css-11inlkn elghiqq0"
    >
      <nav
        class="css-1pdtlkx-PrimaryNav e1befqft0"
      >
        <a
          aria-label="Twitter"
          class="e1befqft1 css-1y78gur e70ypka2"
          data-index="0"
          href="https://twitter.com/raisedadead"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            class="css-1xssqnp e70ypka1"
          >
            <span
              class="css-1sp7cuy e70ypka0"
            >
              <span
                class="css-1fns1v3 e1i5p1da0"
              >
                <i
                  class="fa"
                />
              </span>
            </span>
          </span>
        </a>
        <a
          aria-label="GitHub"
          class="e1befqft1 css-1y78gur e70ypka2"
          data-index="1"
          href="https://github.com/raisedadead"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            class="css-1xssqnp e70ypka1"
          >
            <span
              class="css-1sp7cuy e70ypka0"
            >
              <span
                class="css-1fns1v3 e1i5p1da0"
              >
                <i
                  class="fa"
                />
              </span>
            </span>
          </span>
        </a>
        <a
          aria-label="Medium"
          class="e1befqft1 css-1y78gur e70ypka2"
          data-index="2"
          href="https://blog.raisedadead.com"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            class="css-1xssqnp e70ypka1"
          >
            <span
              class="css-1sp7cuy e70ypka0"
            >
              <span
                class="css-1fns1v3 e1i5p1da0"
              >
                <i
                  class="fa"
                />
              </span>
            </span>
          </span>
        </a>
        <a
          aria-label="freeCodeCamp.org"
          class="e1befqft1 css-1y78gur e70ypka2"
          data-index="3"
          href="https://freeCodeCamp.org/raisedadead"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            class="css-1xssqnp e70ypka1"
          >
            <span
              class="css-1sp7cuy e70ypka0"
            >
              <span
                class="css-1fns1v3 e1i5p1da0"
              >
                <i
                  class="fa"
                />
              </span>
            </span>
          </span>
        </a>
        <a
          aria-label="LinkedIn"
          class="e1befqft1 css-1y78gur e70ypka2"
          data-index="4"
          href="https://linkedin.com/in/mrugeshm"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            class="css-1xssqnp e70ypka1"
          >
            <span
              class="css-1sp7cuy e70ypka0"
            >
              <span
                class="css-1fns1v3 e1i5p1da0"
              >
                <i
                  class="fa"
                />
              </span>
            </span>
          </span>
        </a>
      </nav>
      <h6
        class="css-1xj32bh e19le4zx0"
      >
         
        © 
        2019
         Mrugesh Mohapatra.
      </h6>
    </section>
  </footer>
</div>
`;
